commands/mail.js

```diff
@@ -5,47 +5,21 @@ const fs = require("fs");
 const client = require("@sendgrid/client");
 const postmark = require("postmark");
 const { UpdateMessageStreamRequest } = require("postmark/dist/client/models");
-
-/**
- *
- * @param {} input - if the string is empty, we want to pass it as undefined to the api
- *
- */
-function protectAgainstEmpty(input) {
-  if (input && typeof input === "string" && input === '""') {
-    return undefined;
-  }
-  return input;
-}
-
-/**
- *
- * @param {} input - check to see if the parameter is not empty, then parse before sending to API
- *
- */
-function checkForJson(input) {
-  if (input && typeof input === "string" && input !== '""') {
-    try {
-      return JSON.parse(input);
-    } catch (err) {
-      log.err("JSON was unable to be parsed");
-      process.exit(1);
-    }
-  }
-  return undefined;
-}
+const { checkIfEmpty, unsetField, checkParameters } = require("./../libs/utilities");
```

Comment:
Could we add a comment here that might help beginners understand how these methods are used?